otter/convergence/service.py

```diff
@@ -174,6 +175,7 @@ def execute_convergence(tenant_id, group_id, build_timeout,
 
     worst_status, reasons = yield _execute_steps(steps)
 
+    cache = cache_class(tenant_id, group_id)
```

Comment:
This seemed weird to me, and for a while I couldn't figure out why. I think I figured it out: I think we should have an UpdateServersCache intent, just like the other intents in otter.models.intents like ModifyGroupState.

- we'd be able to get rid of this parameterized `cache_class`
- we'd be able to get rid of the fake `Cache` class entirely

What do you think?